NSS-Day-Cohort-42/rare-server-media-moguls
# Rp add posts

#### Changes Made
1. defined create_post in request.py in the posts directory and imported in request_handler.py where it is called in the do_POST method 
​
#### Steps to Review
1. Checkout this branch locally and run the application.
2. check that posts are saving to the database

```
git fetch --all
git checkout rp-addPost
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user clicks on `New Post`, they should be redirected to a the PostForm page
> When user enters information into the form fields and clicks `Save`, the post database should be updated with new post object and user should be redirected to the PostDetails page to view the post they just created.
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
